rosbag2_storage_mcap/src/mcap_storage.cpp

```diff
@@ -36,7 +36,7 @@
 #endif
 #endif
 
-#include <mcap/mcap.hpp>
+#include <mcap_vendor/mcap/mcap.hpp>
```

Comment:
This shouldn't be necessary; whatever we do here, it should look like we are using a "packaged" version of the library.  So something in the ament exports above isn't right.  If you don't want to deal with the include directory change in this PR we can defer it, but we do need to do that before we merge this package into the core.

Reply:
@clalancette Good point. I Agree. 
I've found how to fix it.